datahub-frontend/test/app/ApplicationTest.java

```diff
@@ -140,8 +144,12 @@ public void testOpenIdConfig() {
   public void testHappyPathOidc() throws InterruptedException {
     browser.goTo("/authenticate");
     assertEquals("", browser.url());
+
     Cookie actorCookie = browser.getCookie("actor");
-    assertEquals("urn:li:corpuser:testUser@myCompany.com", actorCookie.getValue());
+    assertEquals(TEST_USER, actorCookie.getValue());
+
+    Cookie sessionCookie = browser.getCookie("PLAY_SESSION");
```

Comment:
I was so close to checking the session cookie here. Idk why I didn't since I believe this would have detected the missing information in the session.

Reply:
Nice. Thanks for adding.